src/components/ha-selector/ha-selector-media.ts

```diff
@@ -100,7 +100,7 @@ export class HaMediaSelector extends LitElement {
               .disabled=${this.disabled}
               .helper=${this.helper}
               .required=${this.required}
-              include-domains='["media_player"]'
+              .includeDomains=${["media_player"]}
```

Comment:
Can we move the array to a const created outside the render function, so it is not generated again on every render, and causes a rerender on the child too